apps/ledger-live-desktop/src/renderer/screens/platform/v2/Catalog/Card/Minimum.tsx

```diff
@@ -4,11 +4,15 @@ import { Logo } from "./Logo";
 import { PropsRaw } from "./types";
 import { useCard } from "./hooks";
 import { Container, Subtitle } from "./Layout";
+import { useSelector } from "react-redux";
+import { languageSelector } from "~/renderer/reducers/settings";
 
 export function MinimumCard(props: PropsRaw) {
   const { disabled, onClick } = useCard(props);
   const { manifest } = props;
 
+  const rtf = new Intl.RelativeTimeFormat(useSelector(languageSelector));
+
```

Comment:
I would put the whole thing in a `useMemo` as you don't know how expensive the call can be to create the `rtf` and same for the call to `format`
```suggestion
  const lang = useSelector(languageSelector);
  const usedAt = useMemo(() => {
    const rtf = new Intl.RelativeTimeFormat(lang);
    return rtf.format(-manifest.usedAt.diff, manifest.usedAt.unit);
  }, [lang]);
```